scrape/__init__.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+from typing import List
+
+import requests
+from bs4 import BeautifulSoup
+
+from objects.article import Article
+
+URL = "https://news.ycombinator.com/newest"
```

Comment:
Maybe we can move global vars into a class-global var? Since there is a ScrapeManager class